CMakeLists.txt

```diff
@@ -572,7 +572,7 @@ endif()
 
 # Look for OpenGL here before we potentially switch to looking for static libs.
 if (BUILD_CLIENT)
-    if (LINUX)
+    if (LINUX OR FREEBSD)
```

Comment:
Does it really have this Legacy vs. GLVND thing?

Reply:
I expect FreeBSD to be exactly like Linux on that part: they both use Mesa as the de facto standard implementation, not only for drivers, but for the library itself.

The only other non-Mesa OpenGL implementation available on FreeBSD is likely the proprietary Nvidia one for their own proprietary GL driver, and GLVND was actually initiated by Nvidia (because having a vendor-neutral dispatch library only makes sense if there exist more than one vendor library available, and on Linux and FreeBSD, the only two ones are likely Mesa… and Nvidia).

Also FreeBSD provides both legacy `/usr/local/lib/libGL.so` and GLVND `/usr/local/lib/libOpenGL.so`.

Reply:
I confirm both link and run.